src/components/Banner/Banner.tsx

```diff
@@ -181,6 +166,21 @@ export const Banner = ({
           <div className={clsx(styles['banner__action'])}>{action}</div>
         )}
       </div>
+
+      {onDismiss && (
```

Comment:
Because the button is absolutely positioned, we can move it in the DOM order without affecting the visual positioning.